src/components/MAPI/apps/ClusterDetailAppList.tsx

```diff
@@ -22,6 +22,8 @@ const ClusterDetailAppList: React.FC<IClusterDetailAppListProps> = ({
 }) => {
   const [activeIndex, setActiveIndex] = useState(-1);
 
+  const resetActiveIndex = () => setActiveIndex(-1);
```

Comment:
This resets the index so the next app that takes the index of the uninstalled app isn't unexpectedly set to be the active (expanded) app list item